pkg/state/state_rdma_shared_device_plugin_test.go

```diff
@@ -17,70 +17,45 @@ limitations under the License.
 package state_test
 
 import (
-	"context"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
-	"github.com/go-logr/logr"
-	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/fake"
-
 	mellanoxv1alpha1 "github.com/Mellanox/network-operator/api/v1alpha1"
 	"github.com/Mellanox/network-operator/pkg/state"
 )
 
 var _ = Describe("RDMA Shared Device Plugin", func() {
-
-	var ctx context.Context
-	var rdmaDPState state.State
-	var rdmaDPRenderer state.ManifestRenderer
-	var testLogger logr.Logger
-	var catalog state.InfoCatalog
-	var client client.Client
-	var manifestDir string
+	var t testScope
```

Comment:
for me, this reminds me of the `t *testing.T` in (non ginkgo) unit-tests we see in the the Test<Something> funcs.
which is a bit confusing. any chance to rename to something else ?  (tScope, tscope, ts etc...)

that said, i can live with current name if you really like it.